internal/accounts/contract-add.go

```diff
@@ -86,7 +85,7 @@ func addContract(
 		to,
 		flowkit.NewScript(code, contractArgs, filename),
 		globalFlags.Network,
-		false,
+		services.UpdateExistingContract(false),
```

Comment:
nit: we can avoid Contract since it's clear in the context of this function and the name of the function.
```suggestion
		services.UpdateExisting(false),
```

Reply:
Changed